src/ray/raylet/node_manager.cc

```diff
@@ -1927,10 +2090,13 @@ void NodeManager::HandleTaskBlocked(const std::shared_ptr<LocalClientConnection>
       // Release the CPU resources.
       auto const cpu_resource_ids = worker->ReleaseTaskCpuResources();
       local_available_resources_.Release(cpu_resource_ids);
+      if (USE_NEW_SCHEDULER) {
+        new_resource_scheduler_->AddNodeAvailableResources(
+            client_id_.Binary(), cpu_resource_ids.ToResourceSet().GetResourceMap());
+      }
```

Comment:
This code path should never be hit in direct call mode, so we can remove this.

Reply:
done